### DISCUSSION TITLE
require is not defined in plugin

### DISCUSSION THREAD

**Discussion by slavco86** [nuxt/nuxt#16897] [Questions] [answered]
Trying to integrate Azure application insights with our Nuxt 3 app to monitor requests which Nuxt is making server side.
According to app insights docs - I have to use require in order to initialise the package, which I am trying to do via server side plugin, but like it was mentioned above - also getting an error saying required is not defined
https://www.npmjs.com/package/applicationinsights

Is there a work around?

I have tried to use `import`, but I get `undefined value
```
const appInsights = require('applicationinsights');
console.log (appInsights)  //. <-- undefined
```

**Comment by slavco86** [comment]
I have tried to use `import`, but I get `undefined value
```
const appInsights = require('applicationinsights');
console.log (appInsights)  //. <-- undefined
```

**Comment by TheAlexLichter** [comment, marked as answer]
As mentioned in https://github.com/nuxt/framework/issues/1950, did you try `import * as appInsights from 'applicationinsights'`?

**Reply by slavco86** [reply]
Thank you! It works!
